spec/command/trunk/push_spec.rb

```diff
@@ -163,6 +163,30 @@ def found_podspec_among_files(files)
       end
     end
 
+    describe 'sending the swift version up to trunk' do
+      it 'passes the value to trunk' do
+        Command::Trunk::Push.any_instance.unstub(:update_master_repo)
+        Pod::Command::Trunk.any_instance.expects(:json).returns({})
+
+        Pod::Command::Trunk.any_instance.expects(:auth_headers).returns({})
+
+        response = mock
+        response.expects(:headers).returns('location' => ['http://theinternet.com'])
+
+        cmd = Command.parse(%w(trunk push spec/fixtures/BananaLib.podspec --swift-version=1.1.2))
+
+        example = Pod::Specification.from_file('spec/fixtures/BananaLib.podspec')
+        example.attributes_hash[:pushed_with_swift_version] = '1.1.2'
+
+        cmd.stubs(:validate_podspec)
+        cmd.stubs(:request_url)
+
+        api_route = 'pods?allow_warnings=false'
+        cmd.expects(:request_path).with(:post, api_route, example.to_json, {}).returns(response)
```

Comment:
please actually set what `example.to_json` in the test body -- it makes me nervous that it could not be serialized, and this test wouldn't catch that